crates/loader/src/local/mod.rs

```diff
@@ -297,29 +297,46 @@ impl UrlSource {
 
     /// Gets the data from the source as a byte buffer.
     pub async fn get(&self) -> anyhow::Result<Vec<u8>> {
-        let response = reqwest::get(self.url.clone())
-            .await
-            .with_context(|| format!("Error fetching source URL {}", self.url))?;
-        // TODO: handle redirects
-        let status = response.status();
-        if status != reqwest::StatusCode::OK {
-            let reason = status.canonical_reason().unwrap_or("(no reason provided)");
-            anyhow::bail!(
-                "Error fetching source URL {}: {} {}",
-                self.url,
-                status.as_u16(),
-                reason
-            );
-        }
-        let body = response
-            .bytes()
-            .await
-            .with_context(|| format!("Error loading source URL {}", self.url))?;
-        let bytes = body.into_iter().collect_vec();
+        // TODO: when `spin up` integrates running an app from OCI, pass the configured
```

Comment:
could you add an issue to track this after this PR is merged?

Reply:
Good catch, thanks!

Reply:
@fibonacci1729 I for one had forgotten about this comment when you were doing spin up --from-registry - did you already deal to it, or should we resurrect it as an enhancement request

Reply:
@itowlson I completely missed this. We should about an issue. If you haven't got to it by your today, I'll open an issue during my tomorrow

Reply:
I got in there quick so I could assign it to you.  _flees_  #1188